package.json

```diff
@@ -78,7 +78,7 @@
   "dependencies": {
     "@apollo/client": "3.11.8",
     "@ariakit/react": "0.4.14",
-    "@headlessui/react": "^1.7.17",
+    "@headlessui/react": "^2.2.4",
```

Comment:
Bad idea, they changed a lot of apis and implementations, it could break current deprecated components